Polly/src/scenes/LoginScreen/FBLoginButton.js

```diff
@@ -2,6 +2,24 @@ import React, { Component } from 'react'
 import { View, StyleSheet } from 'react-native'
 import { LoginButton, AccessToken } from 'react-native-fbsdk'
 
+
+function createNewUserRequest(accessToken, props) {
+  return fetch('http://localhost:5000/user/', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      "accessToken": accessToken,
+    }),
+  }).then((response) => response.json())
+    .then((responseJson) => responseJson.token)
+    .then((jsonWebToken) => props.setJSONWebToken(jsonWebToken))
+    .catch((error) => {
+      console.error(error);
+    });
+}
+
```

Comment:
this all should be defined in `src/actions/auth.js.` Then you just import this function in the container and map it as a prop using `mapDispatchToProps` Also, you need to find a way to dynamically configure that hostname because it will be different in prod

Reply:
Also, I would use async/await so you avoid all this `.then()` junk

Reply:
When you say `import this function in the container` are you referring to LoginScreen.js that is rendering this FBLoginButton.js component?

Reply:
yee, just like this:
```
import { setAccessToken } from '../../actions/auth'
```